cfme/infrastructure/provider/__init__.py

```diff
@@ -86,9 +86,7 @@ def __init__(
 
     @cached_property
     def vm_name(self):
-        return version.pick({
-            version.LOWEST: "VMs and Instances",
-            '5.7.1': "Virtual Machines"})
+        return "Virtual Machines"
```

Comment:
If there are no other instances of `def vm_name`, can you turn this into a class attribute?

Reply:
OK, I'll look and turn it if everything is ok

Reply:
I haven't wound any other instance of `def vm_name` in infrastructure/provider.